index.html

```diff
@@ -10,145 +10,110 @@
+                    <div class="secondary text">
+                        <p> I can help you build a product , feature or website Look through some of my work and<br class="sentence-break"> experience! If you like what you see and have a project you need coded,<br class="sentence-break"> 
+                            don't hesitate to contact me.</p>
                     </div>
-                    <!--card three-->
-                    <div class="works-item">
-                        <div class="card-img"><div class="card-detail">
-                            <div class="card-main"><h4>Multi-Post Stories Gain+Glory</h4></div>
-                            <div class="lang-list">
-                                <ul>
-                                    <li><a>Ruby on rails</a></li>
-                                    <li><a>CSS</a></li>
-                                    <li><a>JavaScript</a></li>
-                                    <li><a>HTML</a></li>
-                                </ul>
-                            </div>
-                            <div class="card-button">
-                                <a href="#" target="_blank" rel="noopener">See Project</a>
-                            </div>
-                        </div>
+                    <!--div for social links-->
+                    <div class="socials">
+                        <ul>
+                            <li><a class='social-item' id='one' href='https://twitter.com/TanmayiManku' target="_blank" rel="noopener"><img src="./media/twitter.svg" alt="twitter"></a></li>
+                            <li><a class='social-item' id='two' href='https://www.linkedin.com/in/tanmayi-manku-99195720a/' target="_blank" rel="noopener"><img src="./media/Linkedinicon.svg" alt="linkedin"></a></li>
+                            <li><a class='social-item' id='three' href='https://medium.com/' target="_blank" rel="noopener"><img src="./media/medium.svg" alt="medium"></a></li>
+                            <li><a class='social-item' id='four' href='https://github.com/tan12082001' target="_blank" rel="noopener"><img src="./media/github.svg" alt="github"></a></li>
+                            <li><a class='social-item' id='five' href='#' rel="noopener"><img src="./media/angellist.svg" alt="angellist"></a></li>
+                        </ul>   
                     </div>
+                </section>
+                <section class="down-action">
+                    <a href="#portfolio" ><img src='./media/Disabled.svg' alt='drop-down'></a>
+                </section>
+            </div>
+        </div>
+        
+        <!--works, about me and contact me div -->
+        
+        <div class="works-about-contact">
+
+            <!--my recent works section-->
+            <section class="works" id="portfolio">
+                <div class="works-container">
+                    <div class="works-title">
+                    <h1>My Recent Works</h1>
+                    <hr>
```

Comment:
- [ ] For SEO reason, it's advisable to only have one `h1` tag in html document. Kindly use any of the other header tag (`h2, h3 ...`). 👍